### DISCUSSION TITLE
Should fwupdmgr get-devices --show-all-devices list the 40AY0090EU?

### DISCUSSION THREAD

**Discussion by bdaase** [fwupd/fwupd#4005] [Q&A]
I was wondering whether `fwupdmgr get-devices --show-all-devices` is supposed to list the 40AY0090EU Lenovo dock, even though it has no available firmware updates on the LVFS (yet).

I am mainly asking because it is not listed, but I would like to know its current version (to determine whether it is worth it to create a WindowsToGo drive and update the firmware using that one).

**Comment by superm1** [comment]
What version of fwupd?

**Reply by bdaase** [reply]
1.7.1 (Fedora 35)

**Reply by superm1** [reply]
@mrhpearson can you please confirm? I'm guessing the answer is no support right now as that's a brand new version and it's not showing up.

**Reply by bdaase** [reply]
Yeah, I am aware that they don't want to put updates for it on the lvfs (yet), see https://forums.lenovo.com/t5/Other-Linux-Discussions/ThinkPad-Universal-USB-C-Dock-on-LVFS/m-p/5108464.

Anyway, what I was wondering is whether fwupdmgr is still supposed to list the dock anyway as it does with lots of hardware that is generally in the scope but has no updates available.

**Reply by hughsie** [reply]
I believe I have this dock in my giant box of Lenovo dock hardware. The 40au0065 I actually use in the device tests. I think the only thing that I'd need from Lenovo is permission to add the various VIDs and PIDs into the various .quirk files and then the device would enumerate just fine.

**Reply by bdaase** [reply]
That would be great 👍

**Reply by bdaase** [reply]
@hughsie Did you get the VIDs and PIDs in the meantime?

**Reply by hughsie** [reply]
I still need permission from @mrhpearson or similar.

**Comment by mrhpearson** [comment]
I'm pushing the docking team on a few docking + LVFS issues so I'll add this to the requests. 
I don't think they'll sign off on it unless it's been officially tested internally and right now that's not in their plan....but I'm getting a lot of dock related requests so I'm not giving up on it yet.

Mark
